### PR TITLE
tests for unified ova

### PR DESCRIPTION
**Pull request by maplain**
add tests for unified ova

1. need add more test cases;
2. need verify locally then add them to nightly run.

### CODE REVIEW

File: tests/resources/OVA-Util.robot [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    ${out}=  Run   PACKER_ESX_HOST=${host} PACKER_USER=${user} PACKER_PASSWORD=${password} make ova-release 
+    Log  ${out}
+    @{out}=  Split To Lines  ${out}
+    Should Not Contain  @{out}[-1]  Error
+
+Deploy VIC-OVA To Test Server
+    [Arguments]  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL}  ${datastore}=${TEST_DATASTORE}  ${network}=%{BRIDGE_NETWORK}
+    Run Keyword  Build Unified OVA
+    ${rev}= Run  git rev-parse --short HEAD
+    ${out}=  Run  ovftool bin/vic-1.1.0-${rev}.ova bin/vic-1.1.0-${rev}.ovf
```
Comment (frapposelli) [inline tests/resources/OVA-Util.robot]: Is this necessary? I believe ovftool can deploy directly from an OVA, without this intermediate passage

Comment (maplain) [inline tests/resources/OVA-Util.robot]: yeah, you are right. I just copied this part from Harbor-Util, will get rid of it.

File: tests/resources/OVA-Util.robot [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    ${out}=  Run  ovftool --datastore=${datastore} --name=${ova_target_vm_name} --diskMode=thin --powerOn --X:waitForIp --X:injectOvfEnv --X:enableHiddenProperties --prop:appliance.root_pwd=${test_root_pwd} --prop:appliance.permit_root_login=true --prop:network.ip0=${test_network_ip0} --prop:network.netmask0=${test_network_netmask0} --prop:network.gateway=${test_network_gateway} --prop:network.DNS=${test_network_dns} --prop:network.searchpath=${test_network_searchpath} --prop:network.domain=${test_network_domain} bin/vic-1.1.0-${rev}.ovf 'vi://${user}:${password}@${host}'
+    Log  ${out}
+    @{out}=  Split To Lines  ${out}
+    Should Contain  @{out}[-1]  Completed successfully 
+
+Destroy VIC-OVA VM On Test Server
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.destroy ${name}
+    Log  ${output}
+    Should Be Equal As Integers ${rc}  0
```
Comment (lubronzhan) [inline tests/resources/OVA-Util.robot]: There should be two spaces before `${rc}`

File: tests/resources/OVA-Util.robot [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Build Unified OVA
+    [Arguments]  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL}
+    Run  Remove OVA Artifacts Locally
+    ${out}=  Run   PACKER_ESX_HOST=${host} PACKER_USER=${user} PACKER_PASSWORD=${password} make ova-release 
+    Log  ${out}
+    @{out}=  Split To Lines  ${out}
+    Should Not Contain  @{out}[-1]  Error
+
+Deploy VIC-OVA To Test Server
+    [Arguments]  ${user}=%{TEST_USERNAME}  ${password}=%{TEST_PASSWORD}  ${host}=%{TEST_URL}  ${datastore}=${TEST_DATASTORE}  ${network}=%{BRIDGE_NETWORK}
```
Comment (lubronzhan) [inline tests/resources/OVA-Util.robot]: `${datastore}=${TEST_DATASTORE}` -> `${datastore}=%{TEST_DATASTORE}`

File: tests/resources/OVA-Util.robot [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    Run Keyword  Build Unified OVA
+    ${rev}= Run  git rev-parse --short HEAD
+    ${out}=  Run  ovftool bin/vic-1.1.0-${rev}.ova bin/vic-1.1.0-${rev}.ovf
+    ${out}=  Run  ovftool --datastore=${datastore} --name=${ova_target_vm_name} --diskMode=thin --powerOn --X:waitForIp --X:injectOvfEnv --X:enableHiddenProperties --prop:appliance.root_pwd=${test_root_pwd} --prop:appliance.permit_root_login=true --prop:network.ip0=${test_network_ip0} --prop:network.netmask0=${test_network_netmask0} --prop:network.gateway=${test_network_gateway} --prop:network.DNS=${test_network_dns} --prop:network.searchpath=${test_network_searchpath} --prop:network.domain=${test_network_domain} bin/vic-1.1.0-${rev}.ovf 'vi://${user}:${password}@${host}'
+    Log  ${out}
+    @{out}=  Split To Lines  ${out}
+    Should Contain  @{out}[-1]  Completed successfully 
+
+Destroy VIC-OVA VM On Test Server
+    ${rc}  ${output}=  Run And Return Rc And Output  govc vm.destroy ${name}
```
Comment (lubronzhan) [inline tests/resources/OVA-Util.robot]: `${name}` -> `{ova_target_vm_name}`

**Comment by maplain** [comment]
test for /data repartition depends on vcenter url.

**Comment by frapposelli** [comment]
the `ova-tests` branch has been merged into `vmware/vic` you can move this PR to the main repo

**Pull request closed without merging**